tool/prepare-optional-release.ts

```diff
@@ -0,0 +1,11 @@
+import {getDartSassEmbedded} from './utils';
+
+(async () => {
+  try {
+    const dirName = `${process.env.npm_config_platform}-${process.env.npm_config_arch}`;
```

Comment:
It looks like this still isn't set up to take CLI parameters.

Reply:
Have tried CLI parameters in https://github.com/sass/embedded-host-node/commit/e6c9ea2557a7e341eaa5559689dcea2ca7e25211, which needs to change the way of getting `OS` and `ARCH` from a const variable to a function like `getOS(platform)` and `getArch(cpu)` since the OS and platform are different (OS: linux, macos, windows; platform: linux, darwin, win32), now is using an npm script `prepare-optional` and setup `npm_config_platform` and `npm_config_arch` by `npm run prepare-optional --platform=linux --arch=arm64`, I think it is more simple and clean, and it won't have too much invasive on existing implementations.
But if you insist the CLI parameters way is better, I can change it back to https://github.com/sass/embedded-host-node/commit/e6c9ea2557a7e341eaa5559689dcea2ca7e25211.

Reply:
I think it's to be expected that this will change the logic of downloading packages, since the old code strongly assumed that it would always download packages for the current OS. I'd rather have it *always* take parameters and have `prepare-dev-environment` pass in the parameters for the current OS, actually.

Reply:
Clear enough, changed to CLI parameters way in all `prepare-*` scripts, please review again.